# скорость вниз, [м/с]
float32 velocity_down

# скорость вперед, [м/с]
float32 velocity_forward

#скорость вправо, [м/с]
float32 velocity_right